bazel/emscripten_toolchain/emscripten.BUILD

```diff
@@ -1,6 +1,44 @@
 package(default_visibility = ['//visibility:public'])
 
 filegroup(
-    name = "all",
-    srcs = glob(["**"]),
+    name = "includes",
+    srcs = glob([
+        "emscripten/cache/sysroot/include/c++/v1/**",
+        "emscripten/cache/sysroot/include/compat/**",
+        "emscripten/cache/sysroot/include/**",
+        "lib/clang/15.0.0/include/**",
+    ]),
+)
+
+filegroup(
+    name = "compiler_files",
+    srcs = [
+        "emscripten/emcc.py",
+        "bin/clang",
+        "bin/clang++",
```

Comment:
On windows all these executables need to have the `.exe` extension. A separate BUILD file specifically for windows may be necessary or maybe using `build_file_content` and template it.

Reply:
Thank you for this hint! I've taken the templated `build_file_content` approach.